pkg/authorization/authorizer/scope/converter.go

```diff
@@ -178,37 +179,37 @@ func (userEvaluator) Describe(scope string) (string, string, error) {
 	}
 }
 
-func (userEvaluator) ResolveRules(scope, namespace string, clusterPolicyGetter authorizationlister.ClusterPolicyLister) ([]authorizationapi.PolicyRule, error) {
+func (userEvaluator) ResolveRules(scope, namespace string, _ rbaclisters.ClusterRoleLister) ([]rbac.PolicyRule, error) {
 	switch scope {
 	case UserInfo:
-		return []authorizationapi.PolicyRule{
-			{Verbs: sets.NewString("get"), APIGroups: []string{userapi.GroupName, userapi.LegacyGroupName}, Resources: sets.NewString("users"), ResourceNames: sets.NewString("~")},
+		return []rbac.PolicyRule{
```

Comment:
> I'll change it now

Thanks. I don't look in here often so I won't make an issue of it, but are you sure you want to have so many short lines instead of putting them on a single line like we do in the bootstrap files?